debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/LogMinerMetricsMXBean.java

```diff
@@ -147,6 +147,21 @@
      */
     long getNetworkConnectionProblemsCounter();
 
+    /**
+     * @return the total number of milliseconds used to parse DDL/DML statements
+     */
+    long getTotalParseTime();
```

Comment:
Those are the exposed MBean attribute names, correct? If so, let's encode the UOM name in the name, as we do it elsewhere (e.g. "SnapshotDurationInSeconds"). The reason being we cannot expose a proper type like `Duration`,  and a consumer should have no ambiguity over the UOM.